webpack.config.js

```diff
@@ -25,7 +25,10 @@ const {env} = process;
 const IS_PRODUCTION = env.NODE_ENV === 'production';
 const IS_TEST = env.NODE_ENV === 'test' || env.TEST_SUITE;
 const IS_STORYBOOK = env.STORYBOOK_BUILD === '1';
-const IS_CI = !!env.CI || !!env.TRAVIS;
+// This is used to stop rendering dynamic content for tests/snapshots
+// We want it in the case where we are running tests and it is in CI,
+// this should not happen in local
+const IS_CI = (!!env.CI || !!env.TRAVIS) && !!env.VISUAL_SNAPSHOT_ENABLE;
```

Comment:
Do we really need the `!!` thing?

Reply:
No, we're using it to cast to a boolean, I'd prefer to keep it

Reply:
How about this?

```suggestion
const IS_CI = Boolean((env.CI || env.TRAVIS) && env.VISUAL_SNAPSHOT_ENABLE);
```

Reply:
We're already using `!!` everywhere else though, and not that perf matters too much, but [`!!` performs marginally better than `Boolean()`: ](https://perf.link/#eyJpZCI6IjM5NGExbjFjemY4IiwidGl0bGUiOiJDYXN0aW5nIHRvIGJvb2xlYW4iLCJiZWZvcmUiOiJjb25zdCBkYXRhID0gWy4uLkFycmF5KDEwMDApLmtleXMoKV0iLCJ0ZXN0cyI6W3sibmFtZSI6IiEhIiwiY29kZSI6ImZvciAobGV0IGkgaW4gZGF0YSkge1xuICAhIWZhbHNlXG59IiwicnVucyI6WzE4MDAwLDEyMDAwLDEzMDAwLDgwMDAsMTAwMCwxMDAwLDIwMDAsNjAwMCwxMTAwMCwxMjAwMCw2MDAwLDkwMDAsMTMwMDAsMTAwMCwyMDAwLDYwMDAsOTAwMCwxMzAwMCwzMDAwLDYwMDAsMTAwMCwxMjAwMCwxMjAwMCwxMDAwLDE0MDAwLDYwMDAsMjAwMCwxMDAwLDEwMDAsMzAwMCwxODAwMCwxMDAwLDE3MDAwLDE2MDAwLDEyMDAwLDcwMDAsNTAwMCwxOTAwMCwxNDAwMCwzMDAwLDQwMDAsMTAwMCwxMDAwLDUwMDAsNTAwMCw4MDAwLDIwMDAsMTEwMDAsMTYwMDAsMTAwMCw4MDAwLDUwMDAsMTAwMDAsMTAwMCw1MDAwLDUwMDAsNjAwMCwxNTAwMCw5MDAwLDQwMDAsMTkwMDAsMTEwMDAsMTgwMDAsMTAwMCwxMDAwLDEzMDAwLDcwMDAsODAwMCw4MDAwLDE3MDAwLDEwMDAwLDE2MDAwLDE5MDAwLDMwMDAsMTcwMDAsMTYwMDAsMTgwMDAsNTAwMCwzMDAwLDE4MDAwLDE3MDAwLDE4MDAwLDE0MDAwLDMwMDAsMTAwMCw1MDAwLDEwMDAsNzAwMCwxNjAwMCw1MDAwLDEzMDAwLDYwMDAsMTkwMDAsMTkwMDAsMTAwMCw2MDAwLDgwMDAsMTMwMDAsNDAwMCw0MDAwXSwib3BzIjo4NDcwfSx7Im5hbWUiOiJCb29sZWFuIiwiY29kZSI6ImZvciAobGV0IGkgaW4gZGF0YSkge1xuICBCb29sZWFuKGZhbHNlKVxufSIsInJ1bnMiOlsxMDAwLDIwMDAsOTAwMCwxMDAwLDExMDAwLDQwMDAsMTIwMDAsMTAwMCwxMDAwMCw4MDAwLDEwMDAsMTAwMCw2MDAwLDQwMDAsMjAwMCwxMDAwLDMwMDAsMTIwMDAsMTAwMCwxMDAwLDgwMDAsNTAwMCwxMDAwLDEwMDAwLDEzMDAwLDEyMDAwLDUwMDAsMTAwMDAsMTMwMDAsNTAwMCwxMzAwMCwxMDAwLDEwMDAsMTMwMDAsODAwMCwxMzAwMCwxMDAwLDkwMDAsNTAwMCwyMDAwLDgwMDAsMTAwMCw0MDAwLDEwMDAsMzAwMCw0MDAwLDEwMDAsOTAwMCwxMzAwMCwxMDAwMCw1MDAwLDIwMDAsNjAwMCwxMzAwMCwxMDAwLDYwMDAsMjAwMCwxMTAwMCw3MDAwLDEwMDAwLDEwMDAsMjAwMCwxMTAwMCwxMTAwMCwxMDAwLDEzMDAwLDUwMDAsNDAwMCw2MDAwLDcwMDAsMTAwMCwxMDAwLDEwMDAsMTAwMCwyMDAwLDExMDAwLDEzMDAwLDcwMDAsMTAwMCw3MDAwLDEyMDAwLDE0MDAwLDExMDAwLDEwMDAsNTAwMCwzMDAwLDEwMDAsMjAwMCw3MDAwLDEwMDAsMTAwMDAsMzAwMCwxMDAwLDEwMDAsMTAwMCwxMDAwMCwzMDAwLDEwMDAsMTAwMCwyMDAwXSwib3BzIjo1NTIwfV0sInVwZGF0ZWQiOiIyMDIwLTEwLTE2VDIwOjAwOjU1Ljc0NVoifQ%3D%3D)